dd-sdk-android/src/main/kotlin/com/datadog/android/log/LogAttributes.kt

```diff
@@ -30,7 +30,7 @@ object LogAttributes {
      * This value is extracted from your application's manifest and
      * filled automatically by the [Logger].
      */
-    const val APPLICATION_VERSION: String = "application.version"
+    const val APPLICATION_VERSION: String = "version"
```

Comment:
Should we send it as "version" or "application.version" for logs and traces ?

Reply:
It should be `version` indeed :)